neurodamus/core/_shmutils.py

```diff
@@ -43,7 +43,7 @@ def __set_node_info(MPI):  # TODO: Replace with MPI SHM communicator
         SHMUtil.node_id = MPI.py_scatter(node_info, 0)
         SHMUtil.nnodes = MPI.py_broadcast((node_info[-1] + 1) if MPI.rank == 0 else 0, 0)
 
-        subprocess.call(["/bin/rm", "-rf", shmdir])
+        shutil.rmtree(shmdir, ignore_errors=True)
```

Comment:
@WeinaJi  suggested to leave this as it was. She said that it is faster than calling shutil.rmtree. You can add a comment prevent future developers to change this or you can argue with her

Reply:
ok; interesting; thanks.

Reply:
I added a helper with a comment about `shutils.rmtree` being slow; that should make it more clear in the future.